spk/jackett/Makefile

```diff
@@ -29,24 +30,29 @@ ADMIN_PORT = $(SERVICE_PORT)
 UNSUPPORTED_ARCHS = $(PPC_ARCHS)
 
 # Mono build for unsupported .NET archs
-include ../../mk/spksrc.archs.mk
-ifeq ($(findstring $(ARCH),$(ARMv5_ARCHS) $(i686_ARCHS) $(ARMv7L_ARCHS)),$(ARCH))
+include ../../mk/spksrc.common.mk
+ifeq ($(call version_lt, ${TCVERSION}, 7.0),1)
+ifeq ($(findstring $(ARCH),$(ARMv5_ARCHS) $(i686_ARCHS) $(ARMv7L_ARCHS) $(ARMv7_ARCHS)),$(ARCH))
 DEPENDS = cross/curl cross/$(SPK_NAME)-mono
 SERVICE_SETUP = src/service-setup-mono.sh
-CHANGELOG = "Update Jackett to 0.20.1171"
-LICENSE_FILE = $(WORK_DIR)/Jackett/LICENSE
-SPK_DEPENDS = "mono>3.6"
+SPK_DEPENDS = "mono>=5.8"
+endif
+else
+ifeq ($(findstring $(ARCH),$(ARMv5_ARCHS) $(i686_ARCHS) $(ARMv7L_ARCHS) armada370),$(ARCH))
+DEPENDS = cross/curl cross/$(SPK_NAME)-mono
+SERVICE_SETUP = src/service-setup-mono.sh
+SPK_DEPENDS = "mono>=5.8"
+endif
```

Comment:
some suggestions:
As this section is for DSM >= 7, we must not care about  ARMv5_ARCHS and ARMv7L_ARCHS, those will never support DSM 7.
If we want to provide armada370 packages for DSM 7+ that depend on mono, you have to insert the following code
```
else
   UNSUPPORTED_ARCHS += armada370
```

This is needed to exclude armada370 in the package created for the generic arch ARMv7.
Otherwise it is not possible to add both packages to our repository, as we would have duplicate arch/version combination.

Reply:
Thanks for the review. I've attempted to include your suggestions. Do let me know if I interpreted them correctly.